crates/broker/src/lib.rs

```diff
@@ -118,6 +118,17 @@ pub struct Args {
     /// From the `RetryBackoffLayer` of Alloy
     #[clap(long, default_value_t = 100)]
     pub rpc_retry_cu: u64,
+
+    /// Flag indicating if images should be cached locally
+    ///
+    /// If set, images will be downloaded and stored for reuse.
+    /// Currently this only caches the assessor and set-builder images
+    #[arg(long, action = ArgAction::SetTrue)]
+    pub cache_images: bool,
```

Comment:
This seems unused? Perhaps should the cache_dir variable just be renamed to remove this? Or planned to use this flag in the future?

Reply:
Partly for enabling caching of different things (e.g images, inputs) but also to allow for a default cache dir location for when no `--cache-dir` is passed.